cli/flox-rust-sdk/src/providers/build.rs

```diff
@@ -982,14 +980,21 @@ mod tests {
             {pname}>   - copy a bin directory with 'mkdir $out && cp -r bin $out'
             {pname}>   - copy multiple files with 'mkdir -p $out/bin && cp bin/* $out/bin'
             {pname}>   - copy files from an Autotools project with 'make install PREFIX=$out'
+            {pname}>{}
             {pname}> HINT: The following executables were found outside of '$out/bin':
             {pname}>   - not-bin/hello
             {pname}>   - bin/subdir/executable-in-subdir
-        "#};
-        assert!(
-            output.stderr.contains(&expected_output),
-            "{expected_output}"
-        );
+        "#,
+        // Add a trailing space like this so auto-formatters don't trim trailing
+        // whitespace
```

Comment:
suggestion : add a sentence about the original reason to add this

```suggestion
        // Nix logs always include one space of padding even on empty lines.
        // Add a trailing space like this so auto-formatters don't trim trailing
        // whitespace.
```

or instead of this just use an escaped space above via `\x20`, doesn't require this formatting dance to prevent accidental removal and shows that we expect a literal space there.